app/styles/play/ladder/ladder.sass

```diff
@@ -174,6 +174,19 @@
       background: transparent url(/images/common/code_languages/javascript_icon.png) no-repeat center center
       height: 16px
 
+#tournamentend
+  z-index: 1
+  background-color: #ECE0BB
+  width: 1014px
```

Comment:
Could this be a problem on smaller screen sizes? I believe there is some css for smaller screen sizes here: https://github.com/codecombat/codecombat/blob/6b1c859e91af587dc009fa5087d3bcaf117a23aa/app/styles/play/ladder/ladder.sass#L180